### DISCUSSION TITLE
Passing a lot of data to the webview and back

### DISCUSSION THREAD

**Discussion by patrick91** [react-native-webview/react-native-webview#3330] [Q&A]
Hi there!

I was wondering if there'a different way other than using `injectJavaScript` to pass data to the webview :)

My use case is to pass a Y.JS document, by passing a uint8array, which can get quite big depending on the document so passing it as a string might take some time :)

**Comment by linonetwo** [comment]
You need to do the chunking, see how I do it in https://github.com/tiddly-gittly/TidGi-Mobile/blob/master/src/pages/WikiWebView/useStreamChunksToWebView/streamChunksPreloadScript.ts

https://github.com/tiddly-gittly/TidGi-Mobile/blob/master/src/pages/WikiWebView/useStreamChunksToWebView/SQLiteTiddlersReadStream.ts
